packages/components/psammead-social-embed/src/__snapshots__/index.test.jsx.snap

```diff
@@ -59,17 +116,34 @@ exports[`AmpSocialEmbed should render a notice when the provider is unsupported
 <div
   class="c0"
 >
-  <p>
-    Sorry but we're having trouble displaying this content
-  </p>
   <a
-    href="embed-url"
+    class="c1"
+    href="#skip-unknown-content"
   >
-    View content on unknown
+    Skip unknown content
```

Comment:
Is this the text we are actually using? (Translated)

Reply:
No. Please refer to Storybook. Diffs of snapshots are inherently difficult to read.

Reply:
Not sure how to see that on storybook now? Could you give me an example please? "Skip Twitter content"?

Reply:
@lizcameron has kindly reminded me what this will say on the fallback, same as the enriched embed - Thanks both